generator/src/main/java/org/stjs/generator/AnnotationUtils.java

```diff
@@ -0,0 +1,57 @@
+package org.stjs.generator;
+
+import com.sun.tools.javac.code.Symbol;
+import org.stjs.javascript.annotation.AnnotationConstants;
+
+import javax.lang.model.element.Element;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+
+public class AnnotationUtils {
+    /**
+     * Annotation @JSOverloadName specific utils
+     */
+	public static class JSOverloadName {
```

Comment:
I opted for this pattern to create a principle of namespace, it felt better at usage to do: 

`AnnotationUtils.JSOverloadName.decorate()` instead of `AnnotationUtils.decorateForJSOverloadNameAnnotation()` and so on for all methods.